base/doc/source2e.tex

```diff
@@ -342,6 +342,8 @@
 
  \DocInclude{ltoutput} % Output routine
 
+ \DocInclude{lttagging}% Tagging support
```

Comment:
not an issue for this PR but since we went past a-z-A-Z for source2e files, the sorting of files in the code index is off as a-zA-Zaa-zz doesn't work with the current makeindex style.

Reply:
ahh, good point. Could you open a separate pr that this should be looked at at some point?